src/rio.rs

```diff
@@ -0,0 +1,16 @@
+use fastly::Response;
+
+pub mod application;
+pub mod configuration;
+pub mod error;
+pub mod logging;
+
+pub fn generate_synthetic_response(error_message: String, status_code: u16) -> Response {
```

Comment:
I'm not sure this new file is interesting 

1. it's used only once
2. it uses `println`, but in the place you use it, you already have a logger, so it would be better to use the logger.

I would inline the content directly in `main.rs`